lncfg/db.go

```diff
@@ -47,6 +49,8 @@ const (
 	// and channel state DB.
 	NSChannelDB = "channeldb"
 
+	NSGraphDB = "graphdb"
```

Comment:
niiiice - guess we can do a followup PR to implement this and migrations? Think the migration might be huge tho.

Reply:
i think we might actually not need to ever migrate this since: with the Gossip V2 upgrade, we will start using SQL for new messages & we will start using SQL for legacy channels advertised with the new protocol. So eventually, all the channels in the v1 DB _should_ become zombies as the network migrates to V2 and at that point, we can just delete that DB.

Reply:
Think that would take years🤔

Reply:
yeah indeed. so yeah maybe we will end up doing a migration (maybe directly to SQL land) in a follow up.